### PR TITLE
fix : Elevation of speaker image

### LINKED ISSUE #1964: Elevation of speaker title image
**Actual Behaviour**

The speaker image titles are not elevated
**Expected Behaviour**

The speaker image titles should be elevated a bit for a bit of enhancement of UI

**Screenshots of the issue**

screenshot_20171022-064519

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #1964 

Changes: The tiles of speakers images have been given an elevation to make each tile a surface and distinguishable.

Screenshots for the change: 
screenshot_20171022-194452

### CODE REVIEW

File: android/app/src/main/res/layout/item_speaker.xml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
+    android:elevation="6dp"
```
Comment (scandindian) [inline android/app/src/main/res/layout/item_speaker.xml]: This won't work for lower version APIs instead, put this in a CardView and use cardElevation in that

**Review by scandindian — CHANGES_REQUESTED** [review]
Please see inline comments.

File: android/app/src/main/res/layout/item_speaker.xml (hunk 1 above)
Comment (dreadpool2) [reply android/app/src/main/res/layout/item_speaker.xml]: @heysadboy Okay!

**Comment by dreadpool2** [comment]
@heysadboy Does this look okay? (used the card view)
screenshot_20171023-195129

**Comment by scandindian** [comment]
@dreadpool2 Seems like you haven't pushed the code in this pr, this seems great but it would be great if I could see code too.

**Comment by dreadpool2** [comment]
@heysadboy Yeah Sure!

**Comment by dreadpool2** [comment]
@heysadboy I have pushed the code

**Review by scandindian — APPROVED** [review]

**Comment by dreadpool2** [comment]
@the-dagger ,@iamareebjamal Please review this

**Pull request merged**